Sources/Document/Document.swift

```diff
@@ -333,4 +351,57 @@ public actor Document {
     public nonisolated var debugDescription: String {
         "[\(self.key)]"
     }
+
+    private func isSameElementOrChildOf(_ elem: String, _ parent: String) -> Bool {
+        if parent == elem {
+            return true
+        }
+
+        let nodePath = elem.components(separatedBy: ".")
+        let targetPath = parent.components(separatedBy: ".")
+
+        var result = true
+
+        for (index, path) in targetPath.enumerated() where path != nodePath[safe: index] {
+            result = false
+        }
+
+        return result
+    }
+
+    private func processDocEvent(_ event: DocEvent) {
+        if event.type != .snapshot {
```

Comment:
In yorkie-js-sdk, when the target path is `$`, it is possible to receive snapshot events.
Snapshot events are considered as events that occur on the root.
Is there a reason for omitting this part?

https://github.com/yorkie-team/yorkie-js-sdk/blob/20d44ac2eb31f80863cdab938b86657d7645a389/src/document/document.ts#L466-L469C12

Reply:
I missed it. I added it.